browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -968,6 +968,12 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
       <message name="IDS_CONTINUE_TO_WALLET" desc="Text for continue to wallet button">
         Continue to wallet
       </message>
+      <message name="IDS_RECOVERY_PHRASE_12" desc="12-words recovery phrase Wallet button">
+        I have a 12-words recovery phrase
+      </message>
+      <message name="IDS_RECOVERY_PHRASE_24" desc="24-words recovery phrase Wallet button">
+        I have a 24-words recovery phrase
+      </message>
```

Comment:
I believe these should be `12-word` and `24-word`. cc: @rmcfadden3

Reply:
@mkarolin — yes! should be singular. `12-word` and `24-word`

Reply:
👍 Fixed by commit [`f72134b` (#24943)](https://github.com/brave/brave-core/pull/24943/commits/f72134ba2bb49d5e55b9c43e7dae761eb6f6a13d)